site/content/get-started.md

```diff
@@ -0,0 +1,311 @@
+
+{{< note >}}
+The YAML code in the following sections can be founded in the [cafe-example folder](https://github.com/nginxinc/nginx-gateway-fabric/tree/main/examples/cafe-example) of the GitHub repository.
+{{< /note >}}
+
+---
+
+### Create the application resources
+
+Create the file _cafe.yaml_ with the following contents:
+
+{{< ghcode "https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/refs/heads/main/examples/cafe-example/cafe.yaml">}}
+
+Apply it:
+
+```shell
+kubectl apply -f cafe.yaml
+```
+```text
+deployment.apps/coffee created
+service/coffee created
+deployment.apps/tea created
+service/tea created
+```
+
+Verify that the new pods are in the `default` namespace:
+
+```shell
+kubectl -n default get pods
+```
+```text
+NAME                      READY   STATUS    RESTARTS   AGE
+coffee-6db967495b-dvg5w   1/1     Running   0          80s
+tea-7b7d6c947d-8xmhm      1/1     Running   0          80s
+```
+
+---
+
+### Create Gateway and HTTPRoute resources
+
+Create the file _gateway.yaml_ with the following contents:
+
+{{< ghcode "https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/refs/heads/main/examples/cafe-example/gateway.yaml">}}
+
+Apply it using `kubectl`:
+
+```shell
+kubectl apply -f gateway.yaml
+```
+```text
+gateway.gateway.networking.k8s.io/gateway created
+```
+
+Create the file _cafe-routes.yaml_ with the following contents:
+
+{{< ghcode "https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/refs/heads/main/examples/cafe-example/cafe-routes.yaml">}}
+
+Apply it using `kubectl`:
+
+```shell
+kubectl apply -f cafe-routes.yaml
+```
+```text
+httproute.gateway.networking.k8s.io/coffee created
+httproute.gateway.networking.k8s.io/tea created
+```
+
+You can check that all of the expected services are available using `kubectl get`:
+
+```shell
+kubectl get service --all-namespaces
+```
+```text
+NAMESPACE       NAME            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
+default         coffee          ClusterIP   10.96.18.163    <none>        80/TCP                       2m51s
+default         kubernetes      ClusterIP   10.96.0.1       <none>        443/TCP                      4m41s
+default         tea             ClusterIP   10.96.169.132   <none>        80/TCP                       2m51s
+kube-system     kube-dns        ClusterIP   10.96.0.10      <none>        53/UDP,53/TCP,9153/TCP       4m40s
+nginx-gateway   nginx-gateway   NodePort    10.96.186.45    <none>        80:31437/TCP,443:31438/TCP   3m6s
+```
+
+---
+
+## Test NGINX Gateway Fabric
+
+The cluster was configured with port `8080` as the `containerPort` value, alongside the `nodePort` value of the NodePort service.
+
+Since the NodePort `targetPort` values match the _tea_ and _coffee_ service `port` values, no port forwarding is required.
```

Comment:
These NodePort targetPorts are targeting the NGF service, not the tea and coffee services. All traffic flows through NGF, and NGF then sends to those tea/coffee applications.

Reply:
I think this is a case of phrasing: NGF and NodePort are the targets, not the other way around. I'll rewrite it now.